test/driver-e2e-specs.js

```diff
@@ -77,6 +85,53 @@ describe('FakeDriver - via HTTP', function () {
     });
   });
 
+  describe('cli args handling', function () {
+    it('should recieve user cli args for driver if passed in', async function () {
+      let driver = await wdio({...wdOpts, capabilities: caps});
+      const {sessionId} = driver;
+      try {
+        const {data} = await axios.get(`${baseUrl}/${sessionId}/fakecliargs`);
+        JSON.stringify(data.value).should.eql('{}');
+      } finally {
+        await driver.deleteSession();
+      }
+    });
+  });
+
+  describe('cli args handling for empty args', function () {
+    it('should recieve empty user cli args from a driver if no arguments were passed in', async function () {
+      await serverClose();
+      let args = {port: TEST_PORT, host: TEST_HOST, appiumHome, driverArgs: FAKE_DRIVER_ARGS};
+      await serverStart(args);
+
+      let driver = await wdio({...wdOpts, capabilities: caps});
+      const {sessionId} = driver;
+      try {
+        const {data} = await axios.get(`${baseUrl}/${sessionId}/fakecliargs`);
+        JSON.stringify(data.value).should.eql(FAKE_ARGS);
```

Comment:
if we treated fake args as an object before we don't need to stringify here for comparison